var/spack/repos/builtin/packages/caliper/package.py

```diff
@@ -109,6 +110,7 @@ class Caliper(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("unwind@1.2:1", when="+libunwind")
     depends_on("elfutils", when="+libdw")
     depends_on("variorum", when="+variorum")
+    depends_on("intel-oneapi-vtune", when="+vtune")
```

Comment:
Wondering if the type of the dependency is correct. Do you link to any vtune library?

Reply:
> Wondering if the type of the dependency is correct. Do you link to any vtune library?

Yes, Caliper will link to the ITT instrumentation library which is part of vtune.